### ISSUE TITLE
Adjust scene selection radio buttons to make them more accessible

### ISSUE THREAD

**Issue by zepumph** [phetsims/rutherford-scattering#131]
In the keyboar nav meetings in the last few weeks, we have been working on/discussing designs for what to do with the "atomic" vs "nuclear" scale radio buttons. This is what they look like right now:
![image](https://user-images.githubusercontent.com/6856943/33088228-89a4ea36-ce99-11e7-9476-84b9555ee174.png)

The main problem is that for keyboard nav, radio buttons aren't tabbable between them, you have to use arrow keys. There aren't any ques to convey this to visual keyboard users.

We thought about adding a grouping focus highlight, like for NumberControl:
![image](https://user-images.githubusercontent.com/6856943/33088289-b7332990-ce99-11e7-8e7d-c7d90c94a063.png)

But then @mbarlow12 thought about how this is really a problem with universal design, because the same problem is confusing with mouse users. @emily-phet spoke to this especially with mouse users with some cognitive disability.

Here is a mockup that @oliver-phet worked on:
![image](https://user-images.githubusercontent.com/6856943/33088444-17c3e4c0-ce9a-11e7-9086-420ce32abf77.png)

@terracoda really likes using a checkbox on this, since a two scene selection radio button group is basically the same (conceptually). This is what she did in BASE, where there is just a highlight around both buttons, and you can use spacebar to "toggle" between the two scenes.
![image](https://user-images.githubusercontent.com/6856943/33088563-69e84886-ce9a-11e7-9e75-d45672c0ff6f.png)

tagging all others in the meetings: @amanda-phet @arouinfar @ariel-phet @jessegreenberg

**Comment by zepumph** [comment]
Assigning to @terracoda for design

**Comment by jessegreenberg** [comment]
Group decision for  11/21/17:

For now in rutherford-scattering, we will add a thin pink highlight around the radio buttons when one button has focus. We will do that here for now without adding general support for this. This will get us through publication of this sim. We will have a more broad discussion about this group focus highlight at a later date.

**Comment by zepumph** [comment]
@emily-phet's final thought was that the best way to proceed is with treating these as radio buttons, and try to convey the queues/grouping with design/layout.

**Comment by jessegreenberg** [comment]
Design meeting 11/30/17:

For this sim, we decided to add a thin highlight like our number control to indicate 'group' interaction that is manipulated with arrow keys.  We will have additional discussion about whether this is a long term solution. If a long term solution, we should discuss where and how this is applied.

**Comment by jessegreenberg** [comment]
Additional post-meeting notes:

In this meeting, we had a lengthy discussion about the challenges with scene selection buttons and how they should be represented visually and for AT. We went back and forth but did not yet come to a long term conclusion. Here is a collection of thoughts:

- We like the idea of a 'group highlight' that indicates that arrow keys are to be used to interact with the item.
- We would like consistency between what the item looks like and how the item acts for keyboard interaction so that interaction is intuitive.
- We would like project-wide consistency for UI, so that items that look the same should behave the same for mouse and keyboard across sims.
- We considered making a project wide decision so that elements that act like switches (like scene selections with 2 options), look and act like a toggle switch.
  - There was some concern about this because it will require revisiting sims and redesigning the look of these UI components. We could run into cases where we don't have the the view space to do this or the styling isn't appropriate.

@terracoda @emily-phet @zepumph @ariel-phet @arouinfar @oliver-phet @kathy-phet, I am sure this only scratches the surface, please add if you feel something important is not here.

**Comment by jessegreenberg** [comment]
There was more discussion in an a11y meeting today, notes taken in keyboard navigation design document.

**Comment by jessegreenberg** [comment]
After the meeting today, it sounded like for radio buttons we still like the idea of a group focus highlight, and a first pass at implementation happened in https://github.com/phetsims/scenery/issues/708

@terracoda @emily-phet @arouinfar @amanda-phet @zepumph @oliver-phet can you please review how this looks in
https://www.colorado.edu/physics/phet/dev/html/rutherford-scattering/1.1.0-dev.2/rutherford-scattering_en.html?accessibility

The group highlight around the number control was made a little bigger as well.

**Comment by zepumph** [comment]
At first glance it is hard to parse out the difference between the two boxes. I think that having the grouping highlight for the scene selection buttons (and for the number control to a lesser extent too) could be a bit lighter/less opaque. This will help the two differentiate.

**Comment by oliver-phet** [comment]
It looks to me that the focus highlight on the radio buttons is a pixel or two "tighter" to the group focus highlight.
Here's the number control:
<img width="71" alt="screen shot 2017-12-12 at 3 04 42 pm" src="https://user-images.githubusercontent.com/5090275/33911019-0dcf6006-df4e-11e7-9ff0-5d15db12b320.png">
And the group:
<img width="73" alt="screen shot 2017-12-12 at 3 04 52 pm" src="https://user-images.githubusercontent.com/5090275/33911029-1471b3a0-df4e-11e7-9b36-1e97585aa9b1.png">

Maybe @zepumph comment above could be alleviated if there was a similar gap between both highlights?

**Comment by arouinfar** [comment]
I also find it a bit hard to distinguish the focus highlights around the radio buttons, but I think @oliver-phet's suggestion to increase the padding a bit to be on par with the number control may alleviate the situation. Otherwise, the navigation is feeling good.

**Comment by jessegreenberg** [comment]
From the above comments and comments during a a11y keyboard nav meeting, we would like to dilate the bounds around the outer highlight surrounding the radio buttons.

**Comment by emily-phet** [comment]
I believe this was done - @jessegreenberg can you confirm?
